src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -35,12 +35,6 @@ public class PersonCard extends UiPart<Region> {
     @FXML
     private Label phone;
     @FXML
-    private Label gender;
-    @FXML
-    private Label age;
-    @FXML
-    private Label ethnicity;
-    @FXML
```

Comment:
Add back these fields - causing build and UI error.
Change the label to "ethnic"

<img width="793" alt="image" src="https://github.com/AY2324S1-CS2103T-W09-3/tp/assets/64828959/e729e877-cf76-4208-ae55-eaf85cab919e">